### PR TITLE
combine waitlist and checked out pages

### LINKED ISSUE #6438: Combine Loans and Waitlist pages
We currently have loans and waitlists as separate items in the My Books sidebar:

image

As we add more items to this sidebar (Eg #6427 ), this it's getting a little messy. "Waitlist" isn't as useful anymore for most patrons, so lets combine Loans and Waitlist:

image

### Proposal & Constraints
- [ ] /account/loans should render the waitlist items
- [ ] /account/waitlist should redirect to /account/loans
- [ ] Remove the "Waitlist" link from the sidebar
- [ ] Rename "Checked out" to "Loans"

### Stakeholders
@jimchamp

### PR DESCRIPTION
**Pull request by tangym27**
Closes #6438 

From the issue:
- [x]  /account/loans should render the waitlist items
- [x]  /account/waitlist should redirect to /account/loans
- [x]  Remove the "Waitlist" link from the sidebar
- [x]  Rename "Checked out" to "Loans"

### Technical

- contents of `waitlist.html` is now in the `details-content` subsection of `loans.html`
### Screenshot

image

### Stakeholders

@jimchamp

### CODE REVIEW

File: openlibrary/templates/account/loans.html [hunk 1]
```diff
@@ -38,6 +38,11 @@
 .loan-history {
   margin-top: 1em;
 }
+
+.details-content>header{
+    padding-left: 0px;
+}
```
Comment (tangym27) [inline openlibrary/templates/account/loans.html:44]: Used to match the header of `Books You've Checked Out`

**Comment by mekarpeles** [comment]
@jimchamp  + @tangym27 + @cdrini do you think we might want the sidebar label to be "Loans & Waitlists"? Or do we predict people will know that Waitlists have moved to Loans?

**Review by cdrini — CHANGES_REQUESTED** [review]
Nice! Looks good, @tangym27 :) A few small cleanups!

I was able to test the loans page functions correctly; couldn't find a waitlist item to test on :P Will test later.

**Comment by cdrini** [comment]
@mekarpeles I think it'll be sufficiently discoverable; Having a single page dashboard will be a +1 (Note waitlist is 30th on our most visited GA pages).

**Comment by cdrini** [comment]
@tangym27 And some UX tweaks:

- Can you move the "Looking for a book to borrow" section to the bottom, after the waitlist? So something like this.
image

- The "Loans" section looks kind of funny; can you combine the two sections? Ie something like:
image

File: openlibrary/templates/account/books.html [hunk 2]
```diff
@@ -100,7 +100,7 @@ <h1 class="details-title">
     $if key == 'loans':
       $:render_template('account/loans', logged_in_user, items)
     $elif key == 'waitlist':
-      $:render_template('account/waitlist', logged_in_user)
+      $:render_template('account/loans', logged_in_user, items)
```
[the remark this answers is not in the document]
Comment (tangym27) [reply openlibrary/templates/account/books.html:103]: oh, this is actually really nice to learn about. I saw that there was a `waitlist` key in `mybooks.py` that seemed separate from the `waitlist` key in `books.html` so I didn't want to accidentally remove the one used within `mybooks.py`. thanks!

**Review by cdrini — APPROVED** [review]
Oh sorry for the delay! Missed the update on this one :) This looks great! I can't find a way to test waitlisted books right now, so I'll keep an eye out for any user reports if something's not working :+1: Thank you @tangym27 !

**Pull request merged**